spec/features/monitoring_feature_spec.rb

```diff
@@ -10,6 +10,15 @@
     visit authenticated_root_path
   end
 
+  scenario "It works if there are no minions", js: true do
+    Minion.destroy_all
+    visit authenticated_root_path
+
+    using_wait_time 10 do
+      expect(page).not_to have_content("minion0.k8s.local")
```

Comment:
Since we only care about JS errors not being raised, consider using a more generic assertion and changing the test description to something like:

"javascript doesn raise and error when there are no minions [fixed bug]"